packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -27,7 +29,7 @@ describe("deployments", () => {
 			rest.get(
 				"*/accounts/:accountId/workers/services/:scriptName",
 				(request, response, context) => {
-					expect(["undefined", "somethingElse"]).toContain(
+					expect(["undefined", "test-script-name", "something-else"]).toContain(
```

Comment:
Not specifically aimed at this PR, but this is a bit of a strange expectation check IMO. Does it really add much value?
I would be more comfortable having specific checks within `it()` clauses that make sure we are receiving exactly what we expect in these requests.

Reply:
Yeah I was just following the previous tests, I removed it so the actual error logic gets tested